index.html

```diff
@@ -892,11 +860,42 @@ <h4>Create Proof (eddsa-jcs-2022)</h4>
 is produced as output.
             </p>
 
-            <p>
-The algorithm is the same as the one described in Section
-[[[#create-proof-eddsa-rdfc-2022]]] with the changes outlined in
-Section [[[#eddsa-jcs-2022]]].
-            </p>
+            <ol class="algorithm">
+              <li>
+Set <var>options</var>.<var>@context</var> to
+<var>unsecuredDocument</var>.<var>@context</var>.
+              </li>
+              <li>
+Let |proof| be a clone of the proof options, |options|.
+              </li>
+              <li>
+Let |proofConfig| be the result of running the algorithm in
+Section [[[#proof-configuration-eddsa-jcs-2022]]] with
+|options| passed as a parameter.
+              </li>
+              <li>
+Let |transformedData| be the result of running the algorithm in Section
+<a href="#transformation-eddsa-jcs-2022"></a> with |unsecuredDocument|,
+|proofConfig|, and |options| passed as parameters.
```

Comment:
This just takes `options`, not `proofConfig`, right?

```suggestion
<a href="#transformation-eddsa-jcs-2022"></a> with |unsecuredDocument|
and |options| passed as parameters.
```

Reply:
Yes. I'll fix this with the previous fix.